core-bundle/src/Controller/AbstractController.php

```diff
@@ -72,4 +74,53 @@ protected function getCsrfFormOptions(): array
             'csrf_token_id' => $this->getParameter('contao.csrf_token_name'),
         ];
     }
+
+    /**
+     * Set the cache headers according to the page settings.
+     *
+     * @param Response $response The response object
+     *
+     * @return Response The response object
+     */
+    protected function setCacheHeaders(Response $response, PageModel $pageModel)
+    {
+        // Do not cache the response if caching was not configured at all or disabled explicitly
+        if ((false === $pageModel->cache || $pageModel->cache < 1) && (false === $pageModel->clientCache || $pageModel->clientCache < 1)) {
```

Comment:
```suggestion
        if ($objPage->cache < 1 && $objPage->clientCache < 1)
```

Reply:
Did you use $objPage by accident?

Reply:
Yes, sorry. I have adjusted the suggestion.

Reply:
Fixed in f57ef3ab5